### ISSUE TITLE
[BUG] Scroll assist not working correct on iOS and android when keyboard is shown

### ISSUE THREAD

**Issue by RasmusTraeholt** [kirbydesign/designsystem#2161]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When an input field receives focus and the native keyboard pops up it does not scroll the input field into view on first attempt. Subsequent attempts works fine though.

## Describe how to reproduce the bug
See the video below

## Which Kirby version was used?
5.0.18

## What was the expected behavior?
Selected input is always scrolled into view when keyboard appears

## Add any screenshots

https://user-images.githubusercontent.com/23026215/161723295-3d7dafca-07c2-41e2-b1a3-519df8ce99db.mp4

## Please complete the following information:
- OS: [iOS 15.4, Android API 30]

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/stable/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/stable/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/stable/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [stable branch](https://github.com/kirbydesign/designsystem/tree/stable) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing as this is a duplicate of #2151
